### PR TITLE
SQMAGOPC-139 - Fixed rounding error on shipping tax

### PR DESCRIPTION
**Pull request by roopesalminen**

### CODE REVIEW

File: app/code/community/Op/Checkout/Model/Api/Checkout.php [hunk 1]
```diff
@@ -335,8 +335,9 @@ public function _itemArgs($order)
         }
 
         if (!$order->getIsVirtual()) {
+
             $shippingExclTax = $order->getShippingAmount();
-            $shippingInclTax = $order->getShippingInclTax();
+            $shippingInclTax = $order->getShippingAmount() + $order->getBaseShippingTaxAmount();
```
Comment (MAleksei) [inline app/code/community/Op/Checkout/Model/Api/Checkout.php]: hm... base ? base is in base currency. should this be getShippingTaxAmount ?

Comment (roopesalminen) [reply app/code/community/Op/Checkout/Model/Api/Checkout.php]: Yes, I was actually just changing it

Comment (MAleksei) [reply app/code/community/Op/Checkout/Model/Api/Checkout.php]: Nice.
Test, commit and I will approve then

**Review by MAleksei — APPROVED** [review]
All good

**Pull request merged**